scipy/stats/_hypotests.py

```diff
@@ -1653,33 +1679,34 @@ def cramervonmises_2samp(x, y, method='auto'):
     chosen significance level in this example.
 
     """
-    xa = np.sort(np.asarray(x))
-    ya = np.sort(np.asarray(y))
```

Comment:
Let's take care of input validation first / fail fast.

Reply:
don't we need to keep `np.asarray()`, e.g., if x or y are lists (x and y are array-like)?

Reply:
No, _axis_nanpolicy decorator takes care of that and a lot of other stuff.

Reply:
Thanks @chrisb83!